play-testkit/src/main/scala/org/scalatestplus/play/NewGuiceOneServerPerTest.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2001-2016 Artima, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.scalatestplus.play
+
+import org.scalatest.TestSuite
+import org.scalatestplus.play.guice.GuiceFakeApplicationFactory
+
+// RICH: Replacement for scalatestplus-play's GuiceOneServerPerTest class
+// RICH: When moving into scalatestplus-play library, remember to also update NewGuiceOneServerPerSuite
```

Comment:
I think these are notes tickets like (in this case) https://github.com/akka/akka-grpc/issues/362.

I'm ambivalent. It's obvious to me now, but there's nothing wrong with moving this to issues/issue comments too.

Reply:
I thought that whoever did #362 would look at the source code to move and could read these notes to help them remember to do everything. These are more like subtasks of that issue. I'm happy with them being in the source since I think that issue will act as an overall reminder and these comments give details.

Reply:
Made a note about these comments on #362, if that helps.

Reply:
Yes I agree, and no need to make detailed sub-tickets.

I agree having the comment is useful. It might be nice to specifically link to the issue from the comment, making it easier to cross-reference. It's fine like this as well though.